distribution/docker/solr/README.md

```diff
@@ -0,0 +1,32 @@
+# DDF Compatible Solr Container
+
+This docker image simply builds on top of the official solr image and adds in a few components to make it fully compatible with DDF
+
+## Usage
+
+To use this image, run: `docker container run -d -p 8994:8983 codice/ddf-solr:<ddf-version>`
+
+## Extending
+
+For scenarios where additional cores, plugins, or libaries are required, this image can be used as a source via `FROM codice/ddf-solr:<ddf-version>`
+
+To add an additional ddf compatible core(s) a helper script has been included, run `create-ddf-core [cores...]` in a downstream `Dockerfile`
```

Comment:
✏️ I really don't know much about docker yet and there are 2 DockerFiles in the PR. Can we point users to the right one?

Reply:
✏️ Also, could you provide an example of adding the creation of a core to the README?

Reply:
Users won't ever be interacting with the dockerfiles. They will really only be used by maven and the CI. Users would simply run a container from an image available on the codice dockerhub

I will definitely add a core creation example to the README and probably a docker-compose example with a realistic usage example.